### DISCUSSION TITLE
Checkbox stype filters after search

### DISCUSSION THREAD

**Discussion by Dentrax** [community/community#9719] [Other Feature Feedback, Questions, & Ideas]
Would better to add checkboxes for the scan results. Something like:

- [ ] Go
- [ ] Ruby
- [ ] Markdown
- [ ] ...

So we can easily apply more filters based on the filters that we checked in the result panel.

![Screen Shot 2022-01-06 at 21 40 14](https://user-images.githubusercontent.com/16493751/148433938-d1075687-bc93-4609-969d-ad01ecb56a9d.png)

Benchmark:

![Screen Shot 2022-01-06 at 21 42 30](https://user-images.githubusercontent.com/16493751/148434208-28892f81-1c6f-4189-b7a8-2c167f855e17.png)

**Comment by look** [comment]
To clarify, you'd like these to be checkboxes so you can click multiple so that your search becomes something like `<term> (language:Go OR language:Ruby)`?

**Reply by Dentrax** [reply]
Exactly! ❤️
